### DISCUSSION TITLE
【2023.3.22】community meeting notes

### DISCUSSION THREAD

**Discussion by Stevent-fei** [sealerio/sealer#2133] [General]
hi all, 今晚8点，有咱们sealer的开发者会议，大家有什么议题可以提前准备哦。

**Comment by kakaZhou719** [comment]
0.9.3 Task List  Release review： https://github.com/sealerio/sealer/issues/2126

**Comment by kakaZhou719** [comment]
3月22日 【会议纪要】

1. sealer 0.9.3 Release task list review @kakazhou719
2. sealer后续以ackd镜像作为官方的最佳实践镜像 @Starnop @VinceCui
3. 支持本地运行sealer 镜像以及对应的E2E test等 @zhy76
4. plugins repo需要更新 ackd中的plugin作为最佳实践。@Stevent-fei
5. 同步kxs 的进展，扩展kxs的proposal的计划提交，以及目前k0s的一些优化更新 @starComingup
